catalog/to-consul/resource_test.go

```diff
@@ -1540,12 +1591,13 @@ func createEndpoints(t *testing.T, client *fake.Clientset, serviceName string, n
 	require.NoError(t, err)
 }
 
-func defaultServiceResource(client kubernetes.Interface, syncer Syncer) ServiceResource {
+func defaultServiceResource(client kubernetes.Interface, syncer Syncer, syncLb bool) ServiceResource {
```

Comment:
In our tests, we're using the pattern of just modifying the `ServiceResource` object returned by this function instead of passing in the parameter [[example](https://github.com/hashicorp/consul-k8s/blob/8267732c333b0ba6007e61351f0a0fcb90339090/catalog/to-consul/resource_test.go#L111-L112)]. This prevents this function from growing in the number of parameters and also means you don't need to change every reference to it. Would you mind updating it?

Reply:
sure i didn't notice the other way you are doing it

Reply:
fixed via 6c4a823a110582bf74d2b6f8a69ffe08fc21d8c7